types/react-native-side-menu/index.d.ts

```diff
@@ -0,0 +1,76 @@
+// Type definitions for react-native-side-menu 1.1
+// Project: react-native-side-menu
```

Comment:
```suggestion
// Project: https://github.com/react-native-community/react-native-side-menu#readme
```
I'd suggest this, to help someone decide on update,etc (source-of-the-truth)